core/app/controllers/brightcontent/sessions_controller.rb

```diff
@@ -21,7 +21,19 @@ def create
 
     def destroy
       session[:brightcontent_user_id] = nil
+      session[:return_to] = nil
       redirect_to root_url
     end
+
+    private
+
+    def redirect_to_return_to_or(url, *args)
```

Comment:
This method could be a bit simpler:

- you can fetch and delete from a hash in one go (`session.delete`)
- you don't need to duplicate the `redirect_to ..., *args` call if you introduce a variable or refactor the `if` to an inline expression
- perhaps the `url` argument isn't necessary if we always redirect to `root_url` anyways